helm/examples/azure/README.md

```diff
@@ -17,15 +17,16 @@ This may be okay for experimentation, where you do not care that your files are
 
 The hpcc-azurefile helm chart will provision a new Storage Class and a Persistent Volume Claim for each of the required HPCC storage types.
 Once installed the generated PVC names should be used when installing the HPCC helm chart.
-The values-retained-azurefile.yaml is an example of the HPCC storage settings that should be applied, after the "hpcc-azurefile" helm chart is installed.
+The values-retained-azurefile.yaml is an example of the HPCC storage settings that should be applied, after the "hpcc-azurefile" helm chart is installed. "azurestorage-hpcc-azurefile" name is expected when launching "helm install" to match the PVC name in the values-retained-azurefile.yaml.
```

Comment:
should read "azstorage" name is expected when..
to match the example usage on line 28

Reply:
were you going to change this?

Reply:
changed: name is expected when launching "helm install"